lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -238,6 +244,7 @@ local make_entry = function(opts)
         t.lstat = false
       else
         t.lstat = lstat
+        t.readlink = vim.F.if_nil(vim.loop.fs_readlink(t.value),false)
```

Comment:
I'm thinking let's move this to it's own conditional block and use `fs_realpath` instead.
```lua
    if k == "realpath" then
      t.realpath = vim.F.if_nil(vim.loop.fs_realpath(t.value))
      return t.realpath
    end
```